e2-studio-star-rx72n-firmware/libs/rx_spi_link/src/rx_spi_link.c

```
@@ -0,0 +1,629 @@
+ * @pre soft_size >= data_len * k_bits_per_byte
+ * @post soft_len == data_len * k_bits_per_byte
+ *
+ * @since Version 1.1.0
+ */
+static rx_err_t internal_bytes_to_soft_bits(const uint8_t*  data,
+                                             uint32_t        data_len,
+                                             rx_soft_bit_t*  soft,
+                                             uint32_t        soft_size,
+                                             uint32_t*       soft_len)
+{
+    const uint32_t required = data_len * (uint32_t)k_bits_per_byte;
```

Clang (14.0.6)

[warning] variable 'required' is not initialized

(cppcoreguidelines-init-variables)